web/documentserver-example/ruby/app/models/file_model.rb

```diff
@@ -158,7 +158,8 @@ def get_config
         :user => {  # the user currently viewing or editing the document
           :id => !@user.id.eql?("uid-0") ? @user.id : nil,
           :name => @user.name,
-          :group => @user.group
+          :group => @user.group,
+          :image => @user.avatar ? "#{DocumentHelper.get_server_url(true)}/assets/#{@user.id}.png" : nil
```

Comment:
Looks like you missed the `images` subdirectory in the path.

Reply:
> Looks like you missed the `images` subdirectory in the path.

Everything is correct and works. Your option will not work, even if you add “images” for better code readability.